roles/fbc_catalog/tasks/add-bundle.yml

```diff
@@ -27,6 +27,9 @@
   shell:
     cmd: >
       skopeo inspect
+      {% if partner_creds | length %}
+      --authfile {{ partner_creds }}
```

Comment:
Did you mean fbc_opm_auths?

Reply:
For me, the `partner_creds` seems like the correct choice here because we have to access the operator, potentially hosted in the private registry.

Reply:
If `fbc_opm_auths` was not meant in here and `partner_creds` was, then this is adding a new set of credentials that need to be documented and follow the variable role naming.

Reply:
the DOCKER_CONFIG  + '/config.json' contains the credentials file required for skopeo. I suggest just passing to the auth-file option

```
      {% if lookup('env', 'DOCKER_CONFIG') | length %}
      --authfile {{ lookup('env', 'DOCKER_CONFIG')+'/config.json' }}
      {% endif %}
```

Reply:
just for clarification, we checked the status of the docker config file in the affected lab. The file existed but it was empty.